### PR TITLE
Expose flash messages in API response

### PR DESCRIPTION
**Pull request by MusikAnimal**
Bug: T282135

### CODE REVIEW

**Comment by MusikAnimal** [comment]
See https://phabricator.wikimedia.org/T282135#7174690 for context

File: src/Controller/OcrController.php [hunk 1]
```diff
@@ -193,6 +201,15 @@ private function getApiResponse(array $params): JsonResponse
         $response->setEncodingOptions(JSON_NUMERIC_CHECK);
         $response->setStatusCode(Response::HTTP_OK);
 
+        // Expose flash messages.
+        /** @var FlashBag $flashBag */
+        $flashBag = $this->session->getBag('flashes');
+        // @phan-suppress-next-line PhanUndeclaredMethod
```
Comment (Daimona) [inline src/Controller/OcrController.php]: (FTR, you can avoid the double suppression by adding

```
'@phan-var FlashBag $flashBag';
```

at line 207, as a string literal).

Comment (MusikAnimal) [reply src/Controller/OcrController.php]: Nice! I did not know about that, done. A pity that Phan can't read comments!

Comment (Daimona) [reply src/Controller/OcrController.php]: Yeah, unfortunately that's a hard limitation. Inline comments are stripped by the php-ast extension when creating the AST, so phan doesn't even know that they exist.

**Pull request merged**